Tests/SwiftLintFrameworkTests/CommandTests.swift

```diff
@@ -391,4 +392,18 @@ class CommandTests: XCTestCase {
             []
         )
     }
+
+    func testSuperfluousDisableCommandsDisabledWhenAllRulesDisabled() {
+        XCTAssertEqual(
+            violations(Example("// swiftlint:disable all\n// swiftlint:disable non_existent_rule_name\n")),
```

Comment:
I think it's a bit easier to read when using text blocks:
```suggestion
            violations(Example("""
                // swiftlint:disable all
                // swiftlint:disable non_existent_rule_name
            """)),
```